360-Videos-Metadata/360VideosMetadata.py

```diff
@@ -828,8 +833,32 @@ def main():
                       action="store_true",
                       help="injects spherical metadata into a MP4/WebM file, "
                            "saving the result to a new file")
+    parser.add_option("--over_under",
+                      action="store_true",
+                      help="option to include stereo over under information")
+    parser.add_option("--left_right",
+                      action="store_true",
+                      help="option to include stereo left right information")
+
     (opts, args) = parser.parse_args()
 
+    if (opts.over_under and opts.left_right):
+      parser.error("Cannot be both over-under and left-right.");
```

Comment:
if they're mutually exclusive, why not have a single flag that accepts a list of options by using the "choices" feature? [1]

e.g., --stereo_mode=over-under

[1] https://docs.python.org/2/library/optparse.html#optparse.Option.choices